modules/candidate_list/jsx/columnFormatter.js

```diff
@@ -23,16 +23,52 @@ function formatColumn(column, cell, rowData) {
   }
   if (column === 'Scan Done' && cell === 'Y') {
     return (
-        <td className="scanDoneLink">
-            <a href="#"
-               onClick={loris.loadFilteredMenuClickHandler('imaging_browser',
-                 {pscid: rowData[2]})}
-            >
-                {cell}
-            </a>
+      <td className="scanDoneLink">
+        <a href="#"
+           onClick={loris.loadFilteredMenuClickHandler('imaging_browser',
+             {pscid: rowData[2]})}
+        >
+          {cell}
+        </a>
+      </td>
+    );
+  }
+  if (column === 'Monitoring') {
+    if (cell) {
+      var visits = JSON.parse(cell);
+
+      // var visits = cell.split(",");
+      var vlRow = function(visitRow) {
+        if (visitRow.monitored === 'Y') {
+          return (
+            <div className="bg-success" style={{padding: "0px 10px"}}>
```

Comment:
@jstirling91 I was originally using the same colors as above (the feedback colors), @alisterdev convinced me to use those.